X2PO/points-revisions.json

```diff
@@ -1,7 +1,7 @@
 {
-    "current_revision": "X2PO/mar2024/revision.json",
+    "current_revision": "X2PO/sep2024/revision.json",
     "revision_history": [
-        "X2PO/last/revision.json",
+        "X2PO/sep2024/revision.json",
         "X2PO/mar2024/revision.json"
     ]
```

Comment:
This list should be in ascending order (chronologically)

Reply:
Done